webapp/messages/en.json

```
@@ -13,6 +13,7 @@
     "connect-your-wallet": "Connect your Wallet",
     "copied": "Copied",
     "copy": "Copy",
+    "data-attribution": "Data provided by",
```

Should we use interpolation here to inject the link into the text?

```suggestion
    "data-attribution": "Data provided by {link}",
```